strapi/strapi
# Fix Dynamic Zone Tests

### What does it do?
Fixes the `console.error` and adds a test that was in TODO

### Why is it needed?
To fix the pipeline and improve the tests that we already have

### How to test it?
Execute `yarn test:front --verbose packages/core/admin/admin/src/content-manager/components/DynamicZone/` and see that there are not warnings and all the tests are implemented and pass. 

### Related issue(s)/PR(s)
https://github.com/strapi/strapi/issues/17924
